src/Illuminate/Log/Writer.php

```diff
@@ -372,4 +379,22 @@ public function setEventDispatcher(Dispatcher $dispatcher)
     {
         $this->dispatcher = $dispatcher;
     }
+
```

Comment:
The line you commented on, the `$this->dispatcher` was existing. And it has a short description. I'm confused as to what you were referring.